packages/engine/errors/1155.md

```diff
@@ -0,0 +1,20 @@
+---
+original: "'const' declarations must be initialized."
+excerpt: "Const variables should be given a value straight away."
+---
+
+I don't expect `const` variables to ever change after they're created. 
+Therefore, I don't think it makes sense to create one without an initial value.
+
+This confuses me:
+
+```
+const name:string;
```

Comment:
Could you fix the formatting here and change the formatting so it uses TS? I.e. use:

```ts